Bottom: e267dac0e2270db98de2dcd19c1f3ff6960220d1
Top:    e0b8db38756517cb37063d235b13659a02487154
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:48:58 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 926f0ea..68ec2e7 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1255,7 +1255,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                                 copper_appearance,
                                 NULL));
 
-      break;
+      //break;
     }
 
 
@@ -1267,10 +1267,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
